uktrade/great-cms
# KLS-814 - Show warning text when event video unavailable

This PR adds a warning message to the event details page when a video for an event is no longer available. (videos are removed 4 weeks after the event is completed).

Currently the page gives an error when a video no longer exists for an event, so this PR also updates the details page view to only pass through event video data via context if there is a video related to the event.

**Screenshot**
<img width="1173" alt="Screenshot 2023-07-05 at 18 03 09" src="https://github.com/uktrade/great-cms/assets/22460823/30ce4342-a7aa-4fd1-8d72-2d76124fdb53">

**To test**
- Find the id of an event `make manage shell_plus` - `Event.objects.first().id`
- Go to http://greatcms.trade.great:8020/export-academy/event/<event_id>
- The warning text should appear at the top (assuming there is no video added for the event)

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-814

### Reviewing help


### Housekeeping


### Merging
